sklearn/calibration.py

```diff
@@ -460,7 +486,16 @@ def _more_tags(self):
 
 
 def _fit_classifier_calibrator_pair(
-    estimator, X, y, train, test, supports_sw, method, classes, sample_weight=None
+    estimator,
+    X,
+    y,
+    train,
+    test,
+    supports_sw,
+    method,
+    classes,
+    class_weight=None,
```

Comment:
Is `class_weight` used in this function .. ?

Reply:
It's not needed here indeed, thanks!